src/main/java/org/pitest/maven/PmpMojo.java

```diff
@@ -256,6 +256,11 @@ protected RunDecision shouldRun()
             message = projectName + " is a skipped module";
             theDecision.addReason(message);
         }
+        
```

Comment:
how hard would it be to add a test case for this? Or should we wait for #11 to start adding tests on new code paths ?

Reply:
I could be done but would probably require a lot of mocking, therefore, I would prefer to add tests after #11 has been merged.